### PR TITLE
several function names fixed in devmapper.go

### PR DESCRIPTION
**Pull request by lixiaobing1**
several function names fixed in devmapper.go.

### CODE REVIEW

File: pkg/devicemapper/devmapper.go [hunk 1]
```diff
@@ -792,7 +792,7 @@ func CreateSnapDeviceRaw(poolName string, deviceID int, baseDeviceID int) error
 		if dmSawExist {
 			return ErrDeviceIDExists
 		}
-		return fmt.Errorf("devicemapper: Error running deviceCreate (createSnapDevice) %s", err)
+		return fmt.Errorf("devicemapper: Error running deviceCreate (createSnapDeviceRaw) %s", err)
```
Comment (thaJeztah) [inline pkg/devicemapper/devmapper.go]: Should be with a capital `C`

**Review by thaJeztah — CHANGES_REQUESTED** [review]

**Review by thaJeztah — CHANGES_REQUESTED** [review]
can you squash your commits?

**Comment by lixiaobing10051267** [comment]
yeah, has been squashed.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**